compose/donors.json

```diff
@@ -1,101 +1,310 @@
 [
   {
-    "programId": "DASH-CA",
-    "validWithCurrentDictionary": false,
-    "releaseStatus": "released",
-    "donorId": "DO-DASH-1",
-    "submitterDonorId": "SU-DASH-1",
-    "submittedCoreDataPercent": 10.7,
-    "submittedExtendedDataPercent": 45.6,
-    "registeredNormalSamples": 20,
-    "registeredTumourSamples": 20,
-    "publishedNormalAnalysis": 20,
-    "publishedTumourAnalysis": 30,
-    "alignmentsCompleted": 40,
-    "alignmentsRunning": 50,
-    "alignmentsFailed": 30,
-    "sangerVcsCompleted": 20,
-    "sangerVcsRunning": 30,
-    "sangerVcsFailed": 20,
-    "mutectCompleted": 20,
-    "mutectFailed": 20,
-    "mutectRunning": 20,
-    "processingStatus": 10,
-    "totalFilesCount": 20,
-    "filesToQcCount": 20,
-    "updatedAt": "Tue Apr 28 2020 10:40:37 GMT-0400 (Eastern Daylight Time)",
-    "createdAt": "Tue Apr 28 2020 10:40:37 GMT-0400 (Eastern Daylight Time)"
+    "publishedNormalAnalysis": 2,
+    "publishedTumourAnalysis": 2,
+    "alignmentsCompleted": 4,
+    "alignmentsRunning": 0,
+    "alignmentsFailed": 0,
+    "sangerVcsCompleted": 2,
+    "sangerVcsRunning": 0,
+    "sangerVcsFailed": 0,
+    "mutectCompleted": 2,
+    "mutectRunning": 0,
+    "mutectFailed": 0,
+    "totalFilesCount": 0,
+    "filesToQcCount": 0,
+    "releaseStatus": "NO_RELEASE",
+    "processingStatus": "REGISTERED",
+    "validWithCurrentDictionary": true,
+    "donorId": "DO35082",
+    "submitterDonorId": "PCSI_0111",
+    "programId": "PACA-CA",
+    "submittedCoreDataPercent": 0,
+    "submittedExtendedDataPercent": 0,
+    "registeredNormalSamples": 2,
+    "registeredTumourSamples": 2,
+    "updatedAt": "2021-03-19T15:28:28.388Z",
+    "createdAt": "2020-11-09T21:25:17.717Z",
+    "alignmentFirstPublishedDate": "2020-12-11T19:35:42.567Z",
```

Comment:
mock donors for local development/elasticsearch:
- updated overall formatting - this is what you get back from a real elasticsearch query these days
- added molecular analyses published dates